LondonBritishLibrary/orient/BLorient829fragments.xml

```diff
@@ -589,11 +622,388 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                            
+                        </msIdentifier>
+                        
+                        <msContents>
+                            <summary/>
+                            <msItem xml:id="p9_i1">
+                                <locus target="#25"/>
+                                <title type="complete">Letter to <persName ref="PRS9430Tewodros"/> from an Egyptian official, dated 
+                                    <date when-custom="1271" when="1855" calendar="islamic">1271 H.</date>, in Arabic</title>
+                                <textLang mainLang="ar"/>
+                            </msItem>
+                        </msContents>
+                        
+                        <physDesc>
+                            <objectDesc form="Leaf">
+                                <supportDesc>
+                                    <extent>
+                                        <measure unit="leaf">1</measure>
+                                        <locus target="#25"/>
+                                    </extent>
+                                </supportDesc>
+                            </objectDesc>
+                            
+                            <handDesc>
+                                <handNote script="Arabic" xml:id="h6">
+                                    <date when="1855"/>
+                                </handNote>
+                            </handDesc>
+                        </physDesc>
+                        
+                        <history>
+                            <origin>
+                                <origDate when="1855" evidence="internal-date"/>
+                            </origin>
+                        </history>
+                        
+                        <additional>
+                            <adminInfo>
+                                <recordHist>
+                                    <source>
+                                        <listBibl type="catalogue">
+                                            <bibl>
+                                                <ptr target="bm:Wright1877BM"/>
+                                                <citedRange unit="page">334b</citedRange>
+                                            </bibl>
+                                        </listBibl>
+                                    </source>
+                                </recordHist>
+                            </adminInfo>
+                        </additional>
+                    </msPart>
+                    
+                    <msPart xml:id="p10">
+                        <msIdentifier>
+                            <repository ref="INS00001BL"/>
+                            <collection>Oriental</collection>
+                            <idno>BL Oriental 829, ff. 26-32</idno>
+                            <altIdentifier>
+                                <idno>Wright cat. </idno>
```

Comment:
no number?

Reply:
No, a few parts are only listed in the index and don't have a description...